src/dev/windows/lv_windows_display.c

```diff
@@ -0,0 +1,181 @@
+/**
+ * @file lv_windows_display.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_windows_display.h"
+#if LV_USE_WINDOWS
+
+#include "lv_windows_context.h"
+
+#include <process.h>
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+static unsigned int __stdcall lv_windows_display_thread_entrypoint(
+    void * parameter);
+
+/**********************
+ *  STATIC VARIABLES
+ **********************/
+
+/**********************
+ *      MACROS
+ **********************/
+
+/**********************
+ *   GLOBAL FUNCTIONS
+ **********************/
+
+lv_display_t * lv_windows_create_display(
+    const wchar_t * title,
+    int32_t hor_res,
+    int32_t ver_res,
+    int32_t zoom_level,
+    bool allow_dpi_override,
+    bool simulator_mode)
+{
+    lv_windows_create_display_data_t * data = NULL;
+    lv_display_t * display = NULL;
+
+    do {
+        data = (lv_windows_create_display_data_t *)(malloc(
```

Comment:
I choose malloc instead of lv_malloc for some reasons:

- lv_malloc seems doen't support multi-threading properly
- Memory allocation for Windows related infrastructures is much bigger than the memory pool size which users sets in lv_conf.h especially for the simulation mode, user want to know the actually memory usage for their implementation which will run under the production devices.

Kenji Mouri

Reply:
> Why not use local variables?

Because Windows backend supports multiple display instances.

Kenji Mouri

Reply:
The local static variable is not available because maybe we will change to multiple windows in a window manager thread.

Also, the context need to be existed until the display removed. But Windows window messaging will exit the callback when message processed and wait for the message pump for the next call. So, the standard local variable is not available.

Kenji Mouri

Reply:
OK, I think for another reason. I will update it. Please wait.

Kenji Mouri